scripts/run-swift-format.sh

```diff
@@ -21,10 +21,17 @@ fatal() { error "$@"; exit 1; }
 CURRENT_SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 REPO_ROOT="$(git -C "${CURRENT_SCRIPT_DIR}" rev-parse --show-toplevel)"
 
+FORMAT_COMMAND="lint --strict"
+for arg in "$@"; do
+  if [ "$arg" == "--fix" ]; then
+    FORMAT_COMMAND="format --in-place"
+  fi
+done
+
 SWIFTFORMAT_BIN=${SWIFTFORMAT_BIN:-$(command -v swift-format)} || fatal "❌ SWIFTFORMAT_BIN unset and no swift-format on PATH"
 
-"${SWIFTFORMAT_BIN}" lint \
-  --parallel --recursive --strict \
+"${SWIFTFORMAT_BIN}" $FORMAT_COMMAND \
```

Comment:
I think shellcheck will complain without double quotes here:

```suggestion
"${SWIFTFORMAT_BIN}" "$FORMAT_COMMAND" \
```

Reply:
No this is intentional, the variable has multiple options and with the quotes it _doesn't_ work correctly.

Reply:
Good point, this is an exception. Spellcheck recommends using an array instead: https://www.shellcheck.net/wiki/SC2086

Reply:
Ok fixed, used an array.